examples/bigcommerce/lib/bigcommerce/api.ts

```diff
@@ -27,7 +27,7 @@ export async function getApiToken(): Promise<string> {
       body: JSON.stringify({
         channel_id: 1,
         expires_at: A_WEEK_FROM_NOW,
-        allowed_cors_origins: [`https://${config.bigcommerce.storeName}.mybigcommerce.com`],
+        allowed_cors_origins: [config.deploymentURL],
```

Comment:
You might just want to make the configuration the allowed origins, i.e., `config.bigcommerce.allowedCorsOrigins`. That way instead of setting an empty string in dev, you can do an empty array

Reply:
Also, the allowed origins is really all configuration and we probably want to put any logic of figuring out that config value in the `getConfig` function itself, since that's the point of it, to abstract away getting a valid configuration. The `getApiToken` really shouldn't care about the contents of that array